src/transformers/utils/import_utils.py

```diff
@@ -102,6 +102,12 @@ def _is_package_available(pkg_name: str, return_version: bool = False) -> Union[
 _bitsandbytes_available = _is_package_available("bitsandbytes")
 _eetq_available = _is_package_available("eetq")
 _fbgemm_gpu_available = _is_package_available("fbgemm_gpu")
+try:
+    _flute_available = package_exists = (
+        importlib.util.find_spec("flute") is not None and importlib.metadata.version("flute-kernel") == "0.2.6"
+    )
+except importlib.metadata.PackageNotFoundError:
+    _flute_available = False
```

Comment:
let's do the version check in `is_flute_available` and you shouldn't pin the version like that. We should allow a minimum version.

Reply:
Done